src/ipaperftest/core/main.py

```diff
@@ -92,7 +94,12 @@ def run(self, ctx):
                 output = out(ctx.params['results_output_file'])
                 break
 
+        # Output first to results_output_file / stdout, and then to runner_metadata
+        # After that, build the tarfile will all the logs
         output.render(self.results)
+        output.filename = "runner_metadata/test_result"
+        output.render(self.results)
+        selected_plugin.archive_results(ctx)
```

Comment:
Doesn't this have the potential to blow up if the TODO on line 78 isn't implemented? selected_plugin could be None right?

Reply:
I have updated this, now there's a list of available tests in the click option, ensuring that a valid test is passed here.